docs/src/ci.md

```diff
@@ -587,6 +587,18 @@ tests:
     - npm ci
     - npx playwright test --project=$PROJECT --shard=$SHARD
 ```
+### Google Cloud Build
+
```

Comment:
```suggestion
* langs: js

```

Either make it JS only or add snippets for all the supported languages.

Reply:
Done